content/_data/changelogs/weekly.yml

```diff
@@ -7517,6 +7517,47 @@
   # pull: 4704 (PR title: Add proper title for setup wizard)
   # pull: 4705 (PR title: Update at-since up to May 3, 2020)
 
+- version: '2.236'
+  date: 2020-05-11
+  changes:
+  - type: major bug
+    category: regression
+    pull: 4713
+    issue: 62163
+    authors:
+    - daniel-beck
+    message: |-
+      Make plugin manager work on IE11 again ("regression" in 2.231).
+  - type: rfe
+    category: rfe
+    pull: 4630
+    issue: 61808
+    authors:
+    - daniel-beck
+    message: |-
+      Security hardening: Always round-trip password form control values in an encrypted form, even if not backed by an encrypted <code>Secret</code> field.
+      In case of problems, this can be disabled by setting the system property <code>hudson.util.Secret.AUTO_ENCRYPT_PASSWORD_CONTROL</code> to <code>false</code> on startup.
+  - type: rfe
+    category: rfe
+    pull: 4630
+    issue: 61808
+    authors:
+    - daniel-beck
+    message: |-
+      Security hardening: Always use a placeholder value for password form control values in item related configuration forms when the user is missing Item/Configure permission, even if not backed by an encrypted <code>Secret</code> field.
+      In case of problems, this can be disabled by setting the system property <code>hudson.util.Secret.BLANK_NONSECRET_PASSWORD_FIELDS_WITHOUT_ITEM_CONFIGURE</code> to <code>false</code>.
+  - type: rfe
+    category: developer
+    pull: 4707
+    authors:
+    - oleg-nenashev
+    message: |-
+      Developer: Make the SystemProperties API available to plugins so that the properties could be managed by Jenkins extensions including
```

Comment:
The links are references to be added. One cannot manage System properties by Javadoc

Reply:
I had failed to understand your  phrase "references to be added".  In retrospect, your words were perfectly precise and I failed to parse them.  Sorry about that!